megamek/src/megamek/client/ui/swing/lobby/ChatLounge.java

```diff
@@ -1189,50 +1187,7 @@ public void previewGameBoard() {
         boardPreviewW.setVisible(true);
     }
 
-    /**
-     * Returns the game map as it is currently set in the map settings tab.
-     * When onlyFixedBoards is true, all Generated and Surprise boards are
-     * replaced by empty boards, otherwise they are filled with a generated or
-     * a choice of the surprise maps.
-     */
-    public Board getPossibleGameBoard(boolean onlyFixedBoards) {
```

Comment:
Moved this to "ServerBoardHelper" as it needs to be accessible server-side so that we can reset selected custom deployment zones if map settings change and invalidate any of them.